scripts/install/opt/seagate/cortx/motr/libexec/m0trace_logrotate.sh

```diff
@@ -120,7 +120,7 @@ for motr_logdir in $motr_logdirs ; do
             echo "LOG_DIR is $log_dir"
             
             if [[ $platform = "physical" ]]; then
-                peserve_files=`expr $log_files_max_count - 2`
+                peserve_files=`($log_files_max_count - 2)`
```

Comment:
Same as above. Replace back-tick with $().

Reply:
Fixed

Reply:
I would suggest if there are any more such instances where Expr needs to be converted to $() lets take them in another PR, that ways we won't need to keep re-testing and it will be easier for the reviewers to review.